Makefile.am

```diff
@@ -1,13 +1,13 @@
 # Arbitrary files
 arbitrary_files = \
-	files/sw.txt \
-	files/background_pic.jpg \
-	files/test.txt \
+	showmehow-files/sw.txt \
+	showmehow-files/background_pic.jpg \
+	showmehow-files/test.txt \
```

Comment:
If we adjust `build-flatpak.sh` to not remove `files/`, this may not be necessary.